eXpansion/Framework/PlayersBundle/Model/Base/GamecurrencyQuery.php

```diff
@@ -0,0 +1,674 @@
+<?php
```

Comment:
Same why is it here, 

looks like you generated them twice and once in the wrong bundle ?